src/app/containers/MediaPlayer/helpers/logEmbedSourceStatus/index.test.js

```diff
@@ -0,0 +1,34 @@
+import loggerMock from '#testHelpers/loggerMock';
+import { MEDIA_PLAYER_RESPONSE } from '#lib/logger.const';
+import logEmbedSourceStatus from '.';
+
+const fetchResponse = { status: 200 };
+const url = '/embed/source';
+
+describe('logEmbedSourceStatus', () => {
+  beforeEach(() => {
+    fetch.mockImplementation(() => fetchResponse);
+  });
+
+  async function testLogging(status, logFn) {
+    fetchResponse.status = status;
+    await logEmbedSourceStatus(url);
+    expect(logFn).toHaveBeenLastCalledWith(MEDIA_PLAYER_RESPONSE, {
+      url,
+      status,
+    });
+  }
+
+  it('should log info level on 200', async () => {
+    await testLogging(200, loggerMock.info);
```

Comment:
I would be inclined to separate the set up of the test with the assertion, just for better readability?
It might make sense to duplicate the assertion in each test, rather than abstracting it into a separate function.

e.g.
```
fetchResponse.status = 200;
logEmbedSourceStatus(url);

expect (loggerMock.info).toHaveBeenCalledWith(MEDIA_PLAYER_RESPONSE, {
      url,
      status: 200,
    });
```

Reply:
That's where I started from, but codeclimate kept forcing me to refactor into this form.

Reply:
I've added more status checks and I think the helper function makes the tests easier to read.